plugins/auth-node/src/oauth/OAuthCookieManager.ts

```diff
@@ -152,6 +152,36 @@ export class OAuthCookieManager {
     };
     const req = res.req;
     let output = res;
+
+    // because of a bug in the redirect flow where a failed refresh causes an infinite loop
+    // we should be sure to clear the old token which contains the domain
+    // as of the new update
+    // TODO link to issue
+    const domain = new URL(this.options.callbackUrl).hostname;
+    const chunkedFormatExists = OAuthCookieManager.chunkedCookieExists(
+      req,
+      name,
+    );
+    if (chunkedFormatExists) {
+      for (let chunkNumber = 0; ; chunkNumber++) {
+        const key = OAuthCookieManager.getCookieChunkName(name, chunkNumber);
+        const exists = !!req.cookies[key];
+        if (!exists) {
+          break;
+        }
+        output = output.cookie(key, '', {
```

Comment:
not sure if always sending a cookie to clear the old domain is the right choice and would like some feedback here